packages/components/src/components/NewModal/NewModalBackdrop.tsx

```diff
@@ -27,6 +27,10 @@ const Wrapper = styled.div<{ $alignment: NewModalAlignment; $padding: number }>`
     background: rgb(0 0 0 / 30%);
     align-items: ${({ $alignment }) => mapAlignmentToAlignItems($alignment)};
     justify-content: ${({ $alignment }) => mapAlignmentToJustifyContent($alignment)};
+
+    > * + * {
```

Comment:
Why do you need this? Looks like a magic 😅

Reply:
I couldn't help myself:D It's the deranged owl selector. I'll just use gap:))

Reply:
Yes please :D